src/Jobs/JobsBase.tsx

```diff
@@ -169,6 +172,7 @@ const JobsBase: React.FC<JobsBaseProps> = ({
   const baseColumnCount = 7; // Number of base columns defined in the TableHead
   const customColumnCount = customHeaders ? 1 : 0;
   const totalColumnCount = baseColumnCount + customColumnCount;
+  const [isDrawerOpen, setIsDrawerOpen] = useState(false);
```

Comment:
```suggestion
  const [drawerOpen, setDrawerOpen] = useState(false);
```
Given we are determining if the draw is open by setting this variable could we do a small rename/refactor?

Reply:
I thought in JS using the is- prefix was a common naming convention for bools. Though haven't really adhered to it in other parts of the code. So I guess makes sense to change.